Makefile

```diff
@@ -19,34 +19,32 @@ DIST_EXCLUDES := packaging/package-excludes.txt
 
 VIM_DICT      := en-suse-doc
 ASPELL_DICT   := en_US-suse-doc-aspell.rws
-HUNSPELL_DICT := en_US-suse-doc-hunspell.txt
+HUNSPELL_DICT := en_susedoc.dic
```

Comment:
stupid nit, sorry: Could you change the name and add "US-" a hyphen in "suse-doc" to make it consistent? "en_US-suse-doc.dic"